### PR TITLE
update admx instructions

### PR DESCRIPTION
**Pull request by dawit-simeon**

### CODE REVIEW

**Review by rdorta-simeon — APPROVED** [review]

**Review by ashek-simeon — APPROVED** [review]

File: how-to.md [hunk 1]
```diff
@@ -242,6 +242,13 @@ You can verify the licenses in your tenant [in the Azure Portal](https://portal.
         - Under Configuration Settings, click Computer Configuration on the left tab then select Mozilla > Firefox > Cookies Select “Do not allow preferences to be changed" > on the right panel, select “Enabled" > OK
         - Finish the creation steps
 - Run a Sync to export the newly added ADMX resource and profile to Simeon
+- On Azure DevOps (https://dev.azure.com/) > Tenants > Repos > navigate to Source/Resources/Content/MSGraph/DeviceManagement/GroupPolicyUploadedDefinitionFiles
+- There will be 3 files per ADMX resource that was created. An ADML file, an ADMX file, and a JSON file
+<br />
+<img src="https://raw.githubusercontent.com/simeoncloud/docs/master/assets/images/admx_files.png" width: 300; height: auto;/>
+  - The ADMX and ADML files have the following content: TODO: Add adml file content here
+- Update the contents of the ADMX and ADML files on the baseline repository
```
Comment (david-simeon) [inline how-to.md:250]: What does this mean?

File: how-to.md [hunk 2]
```diff
@@ -242,6 +242,13 @@ You can verify the licenses in your tenant [in the Azure Portal](https://portal.
...
+- You can now successfully deploy the files to a downstream tenant
```
Comment (david-simeon) [inline how-to.md:251]: Do they need to sync to deploy it back to the baseline or is that it?

File: how-to.md [hunk 3]
```diff
@@ -242,6 +242,13 @@ You can verify the licenses in your tenant [in the Azure Portal](https://portal.
         - Under Configuration Settings, click Computer Configuration on the left tab then select Mozilla > Firefox > Cookies Select “Do not allow preferences to be changed" > on the right panel, select “Enabled" > OK
         - Finish the creation steps
 - Run a Sync to export the newly added ADMX resource and profile to Simeon
+- On Azure DevOps (https://dev.azure.com/) > Tenants > Repos > navigate to Source/Resources/Content/MSGraph/DeviceManagement/GroupPolicyUploadedDefinitionFiles
```
Comment (Lance-Simeon) [inline how-to.md:245]: @david-simeon, probably not now, but we should consider getting rid of all DevOps specific instructions. What about clients using GitHub?

File: how-to.md [hunk 4]
```diff
@@ -242,6 +242,13 @@ You can verify the licenses in your tenant [in the Azure Portal](https://portal.
...
+- There will be 3 files per ADMX resource that was created. An ADML file, an ADMX file, and a JSON file
+<br />
+<img src="https://raw.githubusercontent.com/simeoncloud/docs/master/assets/images/admx_files.png" width: 300; height: auto;/>
+  - The ADMX and ADML files have the following content: TODO: Add adml file content here
```
[the remark this answers is not in the document]
Comment (jemal-simeon) [reply how-to.md:249]: This is what the content of the ADML and ADMX files look like in the Simeon repo once they are exported from the tenant

**Comment by Lance-Simeon** [comment]
@jemal-simeon @david-simeon what is the status of this?

**Pull request closed without merging**